### ISSUE TITLE
E038 false positive when FOR /F tokens= and delims= share options string

### ISSUE THREAD

**Issue by tboy1337** [tboy1337/Blinter#33]
## Summary

Blinter reports **E038** on valid `FOR /F` options that combine `tokens=` and `delims=` in the same quoted options string.

## Reproduction

```bat
@echo off
for /f "tokens=1,2 delims=|" %%a in ("url|tag") do (
    echo %%a %%b
)
```

## Actual

- **E038** on the `FOR /F` line: `tokens= contains invalid non-numeric characters`

## Expected

No E038. This is valid `cmd.exe` syntax per `FOR /?`.

## Root cause (suspected)

In `src/blinter/checkers/advanced/vars_syntax.py`, rule E038 uses a regex that matches letters after `tokens=` before the closing quote. Letters in the subsequent `delims=` option are incorrectly treated as part of the `tokens=` value.

## Environment

- Blinter 1.1.6
- Windows 10

**Comment by tboy1337** [comment]
Fixed in https://github.com/tboy1337/Blinter/commit/8f32abaf1100af6b07b1948b205395618a12e8b3
